StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/TextField/TextFieldElement+Card.swift

```diff
@@ -361,4 +361,23 @@ extension TextFieldElement {
             return TextFieldElement.PANConfiguration(cardBrandDropDown: cardBrandDropDown).accessoryView(for: lastFourFormatted, theme: theme)
         }
     }
+    struct USBankNumberConfiguration: TextFieldElementConfiguration {
+        let label = String.Localized.bank_account
+        let bankName: String
+        let lastFour: String
+        let isEditable = false
+
+        private var lastFourFormatted: String {
+            "\(bankName) •••• \(lastFour)"
```

Comment:
I notice we have a space between the bullet points and the last 4, typically for bank accounts we don't do this but I see in the figma we do.

Can you raise the point that typically we don't have this space and confirm if this is intentional?